src/plugin/parsers/light_parser.cpp

```diff
@@ -117,6 +117,87 @@ namespace wmr
 			return; // find_if returns last element even if it is not a positive result
 		}
 		updateTransform( transform, it->second );
+	}
+	void AttributeLightCallback(MNodeMessage::AttributeMessage msg, MPlug& plug, MPlug& other_plug, void* client_data)
+	{
+		// Check if attribute was set
+		if (!(msg & MNodeMessage::kAttributeSet))
+		{
+			return;
+		}
+
+		wmr::LightParser* light_parser = reinterpret_cast<wmr::LightParser*>(client_data);
+
+		MStatus status = MS::kSuccess;
+		MFnLight fn_light(plug.node(), &status);
+		
+		MFnDagNode dagnode = fn_light.parent(0, &status);
+		if (status != MS::kSuccess)
+		{
+			LOGE("Could not get the dag node.");
+			MGlobal::displayError("Error: " + status.errorString());
```

Comment:
LOGE already automatically logs to the Maya script editor log. ::displayError is unnecessary.